etc/jenkins/docker/pom.xml

```diff
@@ -0,0 +1,296 @@
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>mysql</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>${mysql.url}</url>
+                            <outputFileName>${mysql.pkg}</outputFileName>
+                            <outputDirectory>${project.build.directory}/image</outputDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>default</id>
+                        <phase>none</phase>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <images>
+                        <image>
+                            <name>${docker.image}:${project.version}</name>
+                            <build>
+                                <contextDir>${project.build.directory}/image</contextDir>
+                                <buildOptions>
+                                    <squash/>
+                                </buildOptions>
+                            </build>
+                        </image>
+                    </images>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>build-image</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>deploy-image</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>clean-image</id>
+                        <phase>clean</phase>
+                        <goals>
+                            <goal>remove</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Disable resources -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-testResources</id>
+                        <phase>none</phase>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Disable compiler -->
```

Comment:
why not:
```
<maven.main.skip>true</maven.main.skip>
<maven.test.skip>true</maven.test.skip>
```
instead?

Reply:
I had to rewrite everything here after changing packaging to pom. Now rest of the plugins can be resolved using properties.